lib/Migration/Version5000Date20211025124248.php

```diff
@@ -192,17 +201,18 @@ protected function normalizeKey(string $prefixedKey): string {
 		throw new \RuntimeException('Invalid regex pattern');
 	}
 
+	/** @return list<int> */
```

Comment:
should be `int[]` or did i missed to learn a new trick?

Reply:
See https://psalm.dev/docs/annotating_code/type_syntax/array_types/#lists 

int[] is equivalent to `@psalm-return array<array-key, int>` while `list<int>` is a bit more exact and is `@psalm-return array<int, int>`

Reply:
tahnks, have not seen it  on  first  glance, i suspected it to be psalm. But my IDE was also confused ;) `@psalm-return` it should be then :)